nntrainer/tensor/tensor.cpp

```diff
@@ -755,9 +754,9 @@ Tensor Tensor::sum_by_batch() const {
 /**
  * @brief Calculate sum according to the axis.
  */
-Tensor Tensor::sum(unsigned int axis, float alpha, float beta) const {
+Tensor Tensor::sum(unsigned int axis, float alpha) const {
```

Comment:
Q. beta is not supported?

Reply:
can you please elaborate what is the bug here?

Reply:
In sum function the argument beta was supposed to amplify the returned tensor by beta.
But in this case the returned tensor is newly made so the beta always should be zero so beta does not needed.
Correct me if I'm wrong.

Reply:
Right.